.ci/ci_job_flags.sh

```diff
@@ -96,23 +96,21 @@ case "${CI_JOB}" in
 	export KATA_HYPERVISOR="qemu"
 	export KUBERNETES="no"
 	;;
-"CRI_CONTAINERD"|"CRI_CONTAINERD_K8S"|"CC_CRI_CONTAINERD"|"CC_SKOPEO_CRI_CONTAINERD")
+"CRI_CONTAINERD"|"CRI_CONTAINERD_K8S"|"CC_CRI_CONTAINERD"|"CC_SKOPEO_CRI_CONTAINERD"|"CC_CRI_CONTAINERD_K8S"|"CC_SKOPEO_CRI_CONTAINERD_K8S")
 	# This job only tests containerd + k8s
 	init_ci_flags
 	export CRI_CONTAINERD="yes"
 	export CRI_RUNTIME="containerd"
 	export KATA_HYPERVISOR="qemu"
 	case "${CI_JOB}" in
-		"CRI_CONTAINERD_K8S")
+		"CRI_CONTAINERD_K8S"|"CC_CRI_CONTAINERD_K8S"|"CC_SKOPEO_CRI_CONTAINERD_K8S")
```

Comment:
I was trying to run it locally with `export CI_JOB=CC_CRI_CONTAINERD_K8S` but the `.ci/install_kata_image.sh` was failing complaining the `CCV0` variable wasn't set.

It took me while to understand that adding `CC_CRI_CONTAINERD_K8S` and `CC_SKOPEO_CRI_CONTAINERD_K8S` in this case entry is wrong. The way it is, the code path enters here (and the `KUBERNETES` variable is set) then it returns out of the case statement, i.e., the next condition (`"CC_CRI_CONTAINERD"|"CC_SKOPEO_CRI_CONTAINERD"|"CC_CRI_CONTAINERD_K8S"|"CC_SKOPEO_CRI_CONTAINERD_K8S"`) is never evaluated and, thus, `CCV0` is never set.

It needs something like this:
```
diff --git a/.ci/ci_job_flags.sh b/.ci/ci_job_flags.sh
index ec36a93e..b8f02e83 100755
--- a/.ci/ci_job_flags.sh
+++ b/.ci/ci_job_flags.sh
@@ -103,7 +103,7 @@ case "${CI_JOB}" in
        export CRI_RUNTIME="containerd"
        export KATA_HYPERVISOR="qemu"
        case "${CI_JOB}" in
-               "CRI_CONTAINERD_K8S"|"CC_CRI_CONTAINERD_K8S"|"CC_SKOPEO_CRI_CONTAINERD_K8S")
+               "CRI_CONTAINERD_K8S")
                        export KUBERNETES="yes"
                        ;;
                "CC_CRI_CONTAINERD"|"CC_SKOPEO_CRI_CONTAINERD"|"CC_CRI_CONTAINERD_K8S"|"CC_SKOPEO_CRI_CONTAINERD_K8S")
@@ -111,6 +111,7 @@ case "${CI_JOB}" in
                        export CCV0="yes"
                        export UMOCI=yes
                        [[ "${CI_JOB}" =~ SKOPEO ]] && export SKOPEO=yes
+                       [[ "${CI_JOB}" =~ K8S ]] && export KUBERNETES=yes
                        ;;
        esac
        ;;

```

Reply:
Thank you @wainersm! We didn't notice this but have just pushed the above change you suggested so now CCV0 should get set!

Reply:
cool, thanks @GeorginaKin ! I will give it a try again.